src/codec-components/ViewCodec/codec-components.tsx

```diff
@@ -37,6 +38,11 @@ export const CResult: ViewResult = ({ value, inner }) => {
 
 export const CStr: ViewStr = ({ value }) => <div>{value}</div>
 export const CNumber: ViewNumber = ({ value }) => <div>{value}</div>
+export const CBitSeq: ViewBitSeq = ({ value }) => (
+  <div>
+    {value.map((val, idx) => `${val}${idx % 4 === 0 ? "_" : ""}`).join("")}
```

Comment:
I think this might have 2 problems:

- it will start with "#_", right? idx=0 => 0%4 === 0 => append "_"
- It will end with a "_" if there's a multiple of 4 positions.